### PR TITLE
[WIP] - CI Migration Test

### PR DESCRIPTION
**Pull request by 0mdc**
## Motivation and Context

Do not merge - CI migration test PR.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/actions/install_ubuntu_deps/action.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    uses: conda-incubator/setup-miniconda@v3.0.1
+    with:
+      miniconda-version: "latest"
+      python-version: "3.9"
+      activate-environment: "habitat"
+  - name: Install conda and dependencies
+    run: |-
+      echo "Install conda and dependencies"
+      conda install -y pytorch==1.12.1 torchvision==0.13.1 -c pytorch -c nvidia
+      conda install -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis pytest-mock
```
Comment (jturner65) [inline .github/actions/install_ubuntu_deps/action.yml:64]: Please specify the numpy verstion == 1.26.4

File: .github/workflows/install_and_test.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          flake8 \
+          flake8-bugbear==22.6.22 \
+          flake8-builtins \
+          flake8-comprehensions \
+          flake8-return \
+          flake8-simplify \
+          hypothesis==6.29.3 \
+          isort==5.12.0 \
+          mypy \
+          numpy \
```
Comment (jturner65) [inline .github/workflows/install_and_test.yml:31]: Please specify the numpy verstion == 1.26.4

**Comment by 0mdc** [comment]
@jturner65 : This is a test PR. Refer to https://github.com/facebookresearch/habitat-sim/pull/2422

**Comment by 0mdc** [comment]
Closing - NVIDIA configuration for Github Actions is resolved.

CI migration will resume here: https://github.com/facebookresearch/habitat-sim/pull/2422

**Pull request closed without merging**